packages/hash/api/src/model/knowledge/link.model.ts

```diff
@@ -82,22 +89,14 @@ export default class {
     );
   }
 
-  /**
-   * Create a link between a source and a target entity using a specific link
-   * type.
-   *
-   * @param params.accountId the accountId of the account creating the link
-   * @param params.sourceEntityModel the source entity of the link
-   * @param params.linkTypeModel the Link Type of the link
-   * @param params.targetEntityModel the target entity of the link
-   */
-  static async create(
+  private static async createLinkWithoutUpdatingSiblings(
```

Comment:
The reason I didn't originally bother with a JSDoc was because:
1. these methods are private
2. these methods are temporary, they won't be needed once the Graph API handles index creation

Have added the JSDoc in https://github.com/hashintel/hash/pull/1081/commits/b5ce1ccd163d7c4ea3114cd68b8abcf601fe505f

Reply:
Understood, your reasoning makes sense. I just found it a bit hard to follow this file and wouldn't enjoy maintaining it without a few more pointers about why things are how they are